pkg/deployment/agent.go

```diff
@@ -140,6 +140,17 @@ func (a *Agent) Get() *appsv1.DaemonSet {
 								Name:          "admin-http",
 							},
 						},
+						LivenessProbe: &corev1.Probe{
+							Handler: corev1.Handler{
+								HTTPGet: &corev1.HTTPGetAction{
+									Path: "/",
+									Port: intstr.FromInt(int(adminPort)),
+								},
+							},
+							InitialDelaySeconds: 1,
```

Comment:
Wouldn't it be better have a higher value here, as readiness probe triggers first after 1 second - so liveness could be after 5 seconds?

Reply:
yes, that would be better.

Reply:
Ok ping me once the change is made - and WIP removed from the title.

Reply:
updated.